_posts/2018-08-23-make-your-own-cli-with-golang-and-cobra.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+authors: [nick_geudens]
+title: 'Make your own CLI with golang and cobra'
+image: /img/make-your-own-cli-with-golang-and-cobra/banner.jpg
+tags: [Golang,Cobra,CLI,Go]
+category: Development
+comments: true
+---
+
+# Table of contents
+1. [Getting started](#getting-started)
+2. [Adding functionality](#adding-functionality)
+3. [Adding flags](#adding-flags)
+4. [Using environment variables instead of flags](#using-environment-variables-instead-of-flags)
+5. [Taking a file as input](#taking-a-file-as-input)
+6. [That’s it!](#thats-it)
```

Comment:
Probably best to replace all Microsoft Word quotes(`’` by regular ones (`'`).
Same for the double quotes.